### PR TITLE
Grade User bugfix 

### LINKED ISSUE #352: Remove bang line from bash code questions
The bang line `#!..` seems mandatory otherwise the error message is very strange.
This should not be mandatory to create a bash question.

### LINKED ISSUE #353: Sort alphabetical users on Grade User interface
Users should be displayed in an alphabetical order on Grade User UI.

### LINKED ISSUE #356: YAML fields order
YAML fields order should not be alphabetically sorted but according to their relevance.
Also, fields which are empty or default need not be present.

### PR DESCRIPTION
**Pull request by maheshgudi**
1. Students' name in grade user are sorted alphabetically. Closes #353 
2. Fields in Yaml question dump are sorted according to relevance. Closes #356 
3. Adds bash command instead of just making it executable for bash evaluator Closes #352

### CODE REVIEW

**Comment by prathamesh920** [comment]
Looks good to me except that Prabhu Sir mentioned that the bang line should be eliminated, and we can use bash command.
Also, is it possible to test the sequence of the yaml fields?

**Comment by maheshgudi** [comment]
@ankitjavalkar Updated the changelog.
@prathamesh920 Resolved the issue raised by Prabhu sir. Also, it is rather difficult to test the order of yaml fields.

File: CHANGELOG.txt [hunk 1]
```diff
@@ -1,3 +1,18 @@
+=== 0.7.0 (Tentatively 23-10-2017) ===
+
+* Simplified Standard I/O type question error output.
+* Added the facility to search questions using tags.
+* Added the facility to search for user profiles on admin site.
+* Replaced json format with yaml format for downloading and uploading questions.
+* Fixed a bug that returned multiple answerpaper objects for a user.
+* Fixed a bug that allowed anyonymous users access quit exam page.
```
Comment (prathamesh920) [inline CHANGELOG.txt]: *anonymous

**Comment by prathamesh920** [comment]
LGTM

**Pull request merged**